apps/website/get-started/README.md

```diff
@@ -11,7 +11,7 @@ Clarity’s designs stem from continuous exploration and research. These designs
 
 ## Device and Browser Support
 
-<ClrImage src="/images/get-started/device_support.png" width="100%" />
+<ClrImage src="/images/get-started/device_support.png" width="100%" aria-label="Image showing supported browser list that container icons for: IE11, Edge, Chrome, Safari and Firefox."/>
```

Comment:
I think this is supposed to say `that contains icons for: ` instead of `that container icons for:`